README.md

```diff
@@ -254,10 +257,28 @@ let credential = await navigator.credentials.get({
   }
 });
 ```
+
 The browser looks into the credential store and sees that there is a credential this is effective for `example.com` from `login.idp.net`.
 The browser give the user a "credential chooser" UI that allows them to share their account at `login.idp.net` with `example.com`.
 Once the user consents, a link is made and the Promise is resolved with a Credential.
 
+### Scenario 4: User intends to link to an identity provider they are already logged in to, but the relying party cannot provide the origin of
+
+In this scenario, the user is already logged into an identity provider that the relying party is willing to accept, but may not be willing or able to provide the origin of.
+This may be because the relying party trusts a class of identity providers with voluntary membership (e.g. IndieAuth), or because they do not wish to provide a list of acceptable identity providers to the browser (e.g. a consortium with anonymous membership).
```

Comment:
```suggestion
This may be because the relying party trusts a class of identity providers with voluntary membership (e.g., IndieAuth), or because they do not wish to provide a list of acceptable identity providers to the browser (e.g., a consortium with anonymous membership).
```